DATA/EQUIPMENT/commodity_equip.ini

```diff
@@ -1224,6 +1224,28 @@ mass = 0
 hit_pts = 250
 lootable = true
 
+[Commodity]
+nickname = commodity_lots_of_credits
```

Comment:
This is a different item than normal Credits? Why?

Reply:
because it has 100k credits instead of 1k. I have forgot to request "amount" for lootbox plugin otherwise i would drop 100 our current credits

Reply:
Since loot boxes do not allow giving out multiple of the same item (nor can they reward credits directly to the player's bank afaik), I've been told this commodity is a temporary means to grant 100K credits via a single item.